unreleased/drosiek-k8s-passthrough.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: k8sprocessor
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: treat k8s.pod.ip in special way
```

Comment:
```suggestion
note: fix the passthrough mode
```
?

Reply:
Much better. Wording is not my best skill